### PR TITLE
style campus view based on mockup

### LINKED ISSUE #158: Have a direct link to campus events form (step 3)

### LINKED ISSUE #159: Have a direct link to campus location form (step 2)

### LINKED ISSUE #160: Have a direct link to campus details form (step 1)

### LINKED ISSUE #313: Campus view should look like the mockup
1. Details
like page 8 of mockups
2. Location
like page 8 of mockups
3. Important dates
like page 8 of mockups
4. Available qualifications
like page 9 of mockups

### PR DESCRIPTION
**Pull request by boney-bun**
fix #313
fix #158
fix #159
fix #160

### CODE REVIEW

**Comment by boney-bun** [comment]
PIA:
ford3#384

Notice that we haven't implement the map location. it only shows `SRID=4326;POINT (-71.060316 48.432044)` in the gif above.

if the location hasn't provided, it will look like this (i improvise by creating a table. maybe we can change into other thing):
ford3#384_2

it turns out the Lato font doesn't load properly in Chrome, but it's working in Safari and Firefox. i have fixed this problem in the PR.

@pierrealixt feedback?

File: django_project/ford3/templates/campus.html [hunk 1]
```diff
@@ -13,72 +13,118 @@
 <div class='container'>
     <div class='row'>
         <div class='col-md-12'>
-            <h3>{{ campus }}</h3>
+            <h3>{{ campus }} Campus Details Review</h3>
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus.html]: just the campus' name is fine.

File: django_project/ford3/templates/campus.html [hunk 2]
```diff
@@ -13,72 +13,118 @@
...
-            <div class='row'>
-                <div class='col-md-4'>
-                    <p>Tel. Number: {{ campus.telephone }}</p>
-                    <p>Max number of Students per year in Campus: {{ campus.max_students_per_year }}</p>
-                </div>
-                <div class='col-md-4'>
-                    <p>Physical address: {{ campus.physical_address_line_1 }}, {{ campus.physical_address_line_2 }}, {{ campus.physical_address_city }} {{ campus.physical_address_postal_code }}</p>
+            <div class='row py-4'>
+                <div class="col-md-6">
+                    <p><b>Photo of Campus:</b></p>
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus.html]: Hide it for now as we don't ask for a photo.

File: django_project/ford3/templates/campus.html [hunk 3]
```diff
@@ -13,72 +13,118 @@
...
-                <div class='col-md-4'>
-                    <ul>
-                    {% for event in campus.events %}
-                        <li><b>{{ event.name }}</b> <br /> {{ event.date_start }} - {{ event.date_end }}</li>
-                    {% endfor %}
-                    </ul>
+                <div class='col-md-6'>
+                    <p><b>Telephone number:</b> {{ campus.telephone }}</p>
+                    <p><b>Email:</b> {{ campus.email}}</p>
+                    <p><b>Max number of Students per year in Campus:</b> {{ campus.max_students_per_year }}</p>
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus.html]: that should be the same label as in the form.

File: django_project/ford3/templates/campus.html [hunk 4]
```diff
@@ -13,72 +13,118 @@
...
+                    <b>{{ event.name }}:</b> {{ event.date_start }} - {{ event.date_end }}
+                </div>
+                <div class='col-md-6'>
+                    <b>Links to event:</b> {{ event.http_link }}
+                </div>
+            </div>
+            {% endfor %}
+            <div class='row py-4'>
+                <div class='col-md-10 edu-underline'>
+                    <h4 class="mt-3">4. Qualification Information</h4>
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus.html]: `4. Available qualifications` instead of `4. Qualification Information`

**Comment by pierrealixt** [comment]
`#7ACDCF` for the header
`#EFEEE9` for the rows
use grey-background button for `edit` button
remove `show` button
add a link to the qualification name
add the title `Qualification` for the qualification column
hide `Completion rate`
hide `last edited at`

File: django_project/ford3/templates/campus.html [hunk 5]
```diff
@@ -13,72 +13,118 @@
...
+                    <a href="{% url 'edit-campus' provider_id=provider.id campus_id=campus.id  %}?step=campus-dates" class='edu-button edu-button-orange p-0'>Edit</a>
+                </div>
+            </div>
+            {% for event in campus.events %}
+            <div class='row py-3'>
+                <div class='col-md-6'>
+                    <b>{{ event.name }}:</b> {{ event.date_start }} - {{ event.date_end }}
+                </div>
+                <div class='col-md-6'>
+                    <b>Links to event:</b> {{ event.http_link }}
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus.html]: Do not repeat `Links to event`
It is supposed to be a http link so add a `<a></a>`

File: django_project/ford3/templates/campus.html [hunk 6]
```diff
@@ -13,72 +13,118 @@
...
-                                            <a class="col-md-5 edu-button edu-button-orange noplr" href="{% url 'show-qualification' provider_id=provider.id campus_id=campus.id qualification_id=qualification.id %}">Show</a>
-                                            <!-- spacer between buttons-->
-                                            &nbsp;
-                                            <a class="col-md-5 edu-button edu-button-orange noplr" href="{% url 'edit-qualification' provider_id=provider.id campus_id=campus.id qualification_id=qualification.id %}">Edit</a>
-                                        </div>
+                                    <td class="border-right align-middle text-center">N/A</td>
+                                    <td class="border-right align-middle text-center">N/A</td>
+                                    <td class="px-0 py-0 align-middle action-buttons">
+                                            <a class="col-md-5 edu-button edu-button-orange noplr my-1 ml-1 mr-1" href="{% url 'show-qualification' provider_id=provider.id campus_id=campus.id qualification_id=qualification.id %}">Show</a>
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus.html]: do not use `noplr`

File: django_project/core/base_static/css/stylesheet.css [hunk 7]
```diff
@@ -841,12 +841,15 @@ nav#sidebar ul li a:hover {
     border: none;
 }
 
+.table-borderless > thead > tr > th {
+    background-color: #7ACDCF;
```
Comment (pierrealixt) [inline django_project/core/base_static/css/stylesheet.css]: use CSS variables

File: django_project/core/base_static/css/stylesheet.css [hunk 8]
```diff
@@ -841,12 +841,15 @@ nav#sidebar ul li a:hover {
...
+}
+
 .border-right {
     border-right-color: black !important;
 }
 .table-borderless > tbody > tr > td {
-    background-color: var(--c-blue);
-
+    background-color: #EFEEE9;
```
Comment (pierrealixt) [inline django_project/core/base_static/css/stylesheet.css]: use CSS variables

**Comment by boney-bun** [comment]
the UI of b477398 :
ford3#384rev

@pierrealixt questions:
1. not sure about using grey for the edit button. it blend in with the background color
2. do `links to event` always specify http protocol?

**Comment by pierrealixt** [comment]
1.Do not use the `table-striped` class
2. yes it does

**Comment by boney-bun** [comment]
@pierrealixt do we want to get rid of the underline of a link?
ford3#384_rev

**Comment by boney-bun** [comment]
what c9916f7 do:
ford3#384_rev

**Pull request closed without merging**